<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SavePNG</class>
 <widget class="QDialog" name="Dialog_SavePNG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OutputFilename">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OutputFilename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hasPalette">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>72</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>有调色板</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hasColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>72</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>有彩色</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hasAlpha">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>有Alpha通道</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ColorTypeWarning">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>44</y>
     <width>221</width>
     <height>50</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>204</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>204</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>警告:该组合并非标准的PNG组合,可能会导致常规软件无法读取甚至崩溃,除非你知道自己在干什么</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_useCustomColorsList">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>使用自定义颜色表</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_bitDepth">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_bitDepth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>位深</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_SavePNG</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_SavePNG</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
